toolchains/syno-armv7-6.1/Makefile

```diff
@@ -0,0 +1,29 @@
+TC_NAME = syno-armv7
+
+# generic toolchain for arm7hf
+TC_ARCH = armada370 armada375 armada38x alpine armadaxp comcerto2k monaco
```

Comment:
@hgy59  alpine4k is lacking from my point of view:
- https://github.com/SynoCommunity/spksrc/wiki/Architecture-per-Synology-model
- https://github.com/SynoCommunity/spksrc/blob/master/toolchains/syno-alpine-6.1/Makefile#L3